mne_connectivity/base.py

```diff
@@ -483,7 +483,14 @@ def _prepare_xarray(self, data, names, indices, n_nodes, method,
 
         # set method, indices and n_nodes
         if isinstance(indices, tuple):
-            new_indices = (list(indices[0]), list(indices[1]))
+            if all([isinstance(inds, np.ndarray) for inds in indices]):
```

Comment:
No need for intermediate `list` creation
```suggestion
            if all(isinstance(inds, np.ndarray) for inds in indices):
```